libcaliptra/examples/README.md

```diff
@@ -3,15 +3,31 @@
 In this directory you will find a basic example on how to interact with the
 Caliptra C API and adapt it to your target platform.
 
+## Build
+
+Build can be done either in this directory by naming a target, or in the individual interface example directories.
+
+In the examples root, the following is valid:
+
+`$ make help`
+
+`$ make hwmodel-clean`
+
```

Comment:
can TARGET and ACTION be 2 different params ?
e.g make hw-model run

Reply:
I'm sure I could make that work, but I don't think it's worth the effort.

Reply:
I've simplified the makefile and adapted the suggestion to have a command-line variable to pick the target.